controller/src/test/java/io/pravega/controller/store/stream/ZKStreamMetadataStoreTest.java

```diff
@@ -74,4 +79,52 @@ public void testInvalidOperation() throws Exception {
                 store.getActiveSegments(scope, stream1, null, executor),
                 (Throwable t) -> t instanceof IllegalStateException);
     }
+
+    @Test
+    public void testScaleIncidents() throws Exception {
```

Comment:
I think you have changed incident -> record. We could also call it occurrence or instance instead of incident if you prefer any of those.